src/main/java/io/github/bobocodebreskul/server/WebPathValidator.java

```diff
@@ -0,0 +1,45 @@
+package io.github.bobocodebreskul.server;
+
+import io.github.bobocodebreskul.context.exception.WebPathValidationException;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class WebPathValidator {
+
+  static final String PATH_SHOULD_START_WITH_SLASH = "Path '%s' validation failed. Path should start with /.";
+  static final String PATH_SHOULD_NOT_CONTAIN_WHITESPACES = "Path '%s' validation failed. Path should not contain whitespaces.";
+  static final String PATH_SHOULD_NOT_CONTAIN_ASTERISKS = "Path '%s' validation failed. Path should not contain *.";
+  static final String PATH_SHOULD_NOT_CONTAIN_MORE_THAN_ONE_SLASH_SEQUENTIALLY = "Path '%s' validation failed. Path should not contain more than one '/' sequentially.";
+
+  /**
+   * The method verifies that the provided path is valid.
+   *
+   * @param path
+   * @throws WebPathValidationException when path does not start from /
+   * @throws WebPathValidationException when path has white space
+   * @throws WebPathValidationException when path has asterisk
+   * @throws WebPathValidationException when path contain more than one / sequentially
+   */
+  public static void validatePath(String path) {
+    if (!path.startsWith("/")) {
+      log.error(PATH_SHOULD_START_WITH_SLASH.formatted(path));
+      throw new WebPathValidationException(PATH_SHOULD_START_WITH_SLASH.formatted(path));
+    }
+
+    if (path.contains(" ")) {
```

Comment:
This case only covers single space character. Please take a look on whitespace regex character - `\s`, described here [Java 17 Pattern](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/regex/Pattern.html)
In general, it will look like this - `Pattern.compile("\s").matcher(INPUT_STRING).find()`, and `Pattern.compile("\s")` should be assigned to static constant so you don't need to create it every time.
```
private static final Pattern WHITESPACE_SEARCH_PATTERN = Pattern.compile("\s");
...
if (WHITESPACE_SEARCH_PATTERN.matcher(path).find()) {
  // do you work here
  ...
}
```